sentry-sidekiq/spec/sentry/sidekiq/sentry_context_middleware_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "spec_helper"
+
+RSpec.describe Sentry::Sidekiq::SentryContextServerMiddleware do
+  before { perform_basic_setup }
+
+  it "sets user to the current scope from the job" do
+    user = { id: "u1" }
+    job = instance_double("job")
+    allow(job).to receive(:merge)
+    allow(job).to receive(:[])
+    allow(job).to receive(:[]).with("sentry_user").and_return(user)
+
+    expect(Sentry.get_current_scope).to receive(:set_user).with(user).exactly(1).time
```

Comment:
This spec does not work.

Reply:
can you test this feature end-to-end instead?